docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,105 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
```

Comment:
What happens if rsync isn't installed? Is it assumed to be present, and Pulp will crash if not? Will the plugin check for the presence of rsync before publishing, and return an error if rsync isn't present? Something else?